lib/include/tick/array/vector/cblas.h

```diff
@@ -1,6 +1,24 @@
 
 
-#if defined(TICK_CBLAS_AVAILABLE)
+#ifndef LIB_INCLUDE_TICK_ARRAY_VECTOR_CBLAS_H_
+#define LIB_INCLUDE_TICK_ARRAY_VECTOR_CBLAS_H_
+
```

Comment:
I would add
`#if defined(TICK_USE_CBLAS)`
Even if it is only included if `TICK_USE_CBLAS` is defined.